lib/midi-smtp-server.rb

```diff
@@ -383,6 +388,7 @@ def initialize(
       @do_dns_reverse_lookup = do_dns_reverse_lookup.nil? ? true : do_dns_reverse_lookup
 
       # io and buffer settings
+      @io_waitreadable_sleep = io_waitreadable_sleep
```

Comment:
In case that this is NOT important to instantiate the class, it should be handled as others:

```ruby
@io_waitreadable_sleep = io_waitreadable_sleep.nil? ? DEFAULT_IO_WAITREADABLE_SLEEP : io_waitreadable_sleep
```